### PR TITLE
feat: Scheduler Resource CURD and web page

### PR DESCRIPTION
**Pull request by gavin1ee**
add Scheduler Resource CURD and web page

### CODE REVIEW

File: src/Services/Masa.Scheduler.Services.Server/Application/Resources/Queries/SchedulerResourceQuery.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Scheduler.Services.Server.Application.Resources.Queries;
+
+public record SchedulerResourceQuery(SchedulerResourceListRequest Request) : Query<SchedulerResourceListResponse>
```
Comment (MayueCif) [inline src/Services/Masa.Scheduler.Services.Server/Application/Resources/Queries/SchedulerResourceQuery.cs:6]: SchedulerResourcesQuery  or SchedulerResourceListQuery

File: src/Services/Masa.Scheduler.Services.Server/Application/Resources/Queries/SchedulerResourceQuery.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+{
+    public override SchedulerResourceListResponse Result { get; set; } = null!;
```
Comment (MayueCif) [inline src/Services/Masa.Scheduler.Services.Server/Application/Resources/Queries/SchedulerResourceQuery.cs:8]: new() replace null!

File: src/Web/MASA.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor.cs [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+        }
+    }
+
+    private List<SideBarItem> _allProjects = new();
+
+    private List<SideBarItem> _projects = new();
+
+    private List<SchedulerResourceDto> _resources = new();
+
+    private const string PROJECT_PREFIX = "Project_";
```
Comment (MayueCif) [inline src/Web/MASA.Scheduler.Web.Admin/Pages/SchedulerResources/SchedulerResourceFiles.razor.cs:78]: const should be begin position

**Review by MayueCif — CHANGES_REQUESTED** [review]
@gavin1ee

**Review by MayueCif — APPROVED** [review]

**Comment by MayueCif** [comment]
@doddgu LGTM

File: src/Services/Masa.Scheduler.Services.Server/Application/Resources/SchedulerResourceCommandHandler.cs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    [EventHandler]
+    public async Task UpdateHandleAsync(UpdateSchedulerResourceCommand command)
+    {
+        var resourceDto = command.Request.Data;
+
+        var resource = await _repository.FindAsync(r => r.Id == resourceDto.Id);
+
+        if (resource is null)
+        {
+            throw new UserFriendlyException($"The current resource does not exist");
```
Comment (doddgu) [inline src/Services/Masa.Scheduler.Services.Server/Application/Resources/SchedulerResourceCommandHandler.cs:34]: The exception message no need $ for it.

**Review by doddgu — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**